### PR TITLE
fix(opentrons-ai-client): update prompt guide and examples

### PR DESCRIPTION
**Pull request by Elyorcv**
# Overview
Updated prompt guide and template examples.

Closes: AUTH-408

# Test Plan
Please run: 
- `make test-js`

# Changelog
- Changed prompt guide
- Changed examples (4)

# Review requests
Please see the UI and read and try (please read readme how to run dev)

# Risk assessment
Low

### CODE REVIEW

**Comment by koji** [comment]
`docs` is for doc files like readme.md etc.
You need to put project folder inside ().
for this case, the prefix would be fix(opentrons-ai-client)

File: opentrons-ai-client/src/assets/localization/en/protocol_generator.json
Comment (koji) [inline opentrons-ai-client/src/assets/localization/en/protocol_generator.json:1]: We order keys in i18n json files alphabetically

Comment (koji) [reply opentrons-ai-client/src/assets/localization/en/protocol_generator.json:1]: @Elyorcv Did you ask Ed to the new text?

**Comment by koji** [comment]
Need to fix the format 
https://github.com/Opentrons/opentrons/actions/runs/9159828999/job/25181135320?pr=15220

Basically, we need to pass the following commands

```shell
make check-js
make lint-js
make test-js
```

File: opentrons-ai-client/src/assets/localization/en/protocol_generator.json
Comment (Elyorcv) [reply opentrons-ai-client/src/assets/localization/en/protocol_generator.json:1]: No, only Mike confirmed.

File: opentrons-ai-client/src/assets/localization/en/protocol_generator.json [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
+  "reload_page": "To start over and create a new protocol, simply reload the page.",
+  "robot_type": "Robot type: Choose the OT-2 or Opentrons Flex.",
   "robot": "Robot: OT-2.",
   "share_your_thoughts": "Share your thoughts here",
   "side_panel_body": "Write a prompt in natural language to generate a Reagent Transfer or a PCR protocol for the OT-2 or Opentrons Flex using the Opentrons Python Protocol API.",
   "side_panel_header": "Use natural language to generate protocols with OpentronsAI powered by OpenAI",
   "simulator_description": "Once OpentronsAI has written your protocol, type \"simulate\" in the prompt box to try it out.",
-  "tipracks_and_labware": "<span>Tip racks and labware: Use names from the <a>Opentrons Labware Library</a>.</spa>",
   "try_example_prompts": "Stuck? Try these example prompts to get started.",
+  "type_simulate": "Once OpentronsAI has written your protocol, type “simulate” in the prompt box to try it out.",
```
Comment (koji) [inline opentrons-ai-client/src/assets/localization/en/protocol_generator.json]:
```suggestion
  "simulate_description": "Once OpentronsAI has written your protocol, type “simulate” in the prompt box to try it out.",
```

File: opentrons-ai-client/src/assets/localization/en/protocol_generator.json [hunk 2]
```diff
@@ -1,30 +1,35 @@
...
+  "labware_and_tipracks": "Labware and tip racks: Use names from the <a>Opentrons Labware Library</a>.",
+  "liquid_locations": "Liquid locations: Describe where liquids should go in the labware.",
   "loading": "Loading...",
   "login": "Login",
   "logout": "Logout",
-  "make_sure_your_prompt": "Make sure your prompt includes the following:",
-  "metadata": "Metadata: Three pieces of information.",
-  "modules": "Modules: Thermocycler or Temperature Module.",
-  "opentronsai_asks": "OpentronsAI asks you to provide it!",
+  "make_sure_your_prompt": "Write a prompt in a natural language for OpentronsAI to generate a protocol using the Opentrons Python Protocol API v2. The better the prompt, the better the quality of the protocol produced by OpentronsAI. ",
```
Comment (koji) [inline opentrons-ai-client/src/assets/localization/en/protocol_generator.json]:
```suggestion
  "make_sure_your_prompt": "Write a prompt in a natural language for OpentronsAI to generate a protocol using the Opentrons Python Protocol API v2. The better the prompt, the better the quality of the protocol produced by OpentronsAI.",
```

**Comment by Elyorcv** [comment]
> ```shell
> make check-js
> make lint-js
> make test-js
> ```

Thanks, Done

**Review by koji — APPROVED** [review]
lgtm

**Pull request merged**